### DISCUSSION TITLE
Add grid snapping to bracket editor on tournament client.

### DISCUSSION THREAD

**Discussion by Entropy-10** [ppy/osu#17969] [Ideas]
It would be a huge useability improvement if the bracket editor had grid snapping as currently creating a bracket is extremely difficult if you want them to look even. The current solution is to manually edit the bracket.json and mathematically align your bracket which takes a lot of time especially for large tournament formats.

**Comment by peppy** [comment]
Definitely would be nice to have. In the mean time, you can also consider using the zoom functionality to get more precision. We align using the editor and haven't had much trouble getting things looking fine.

**Comment by Entropy-10** [comment]
Addressed by https://github.com/ppy/osu/pull/24308
